client/lib/plans/features-list.js

```diff
@@ -253,6 +259,12 @@ export const FEATURES_LIST = {
 			),
 	},
 
+	[ constants.FEATURE_ADVANCED_SEO_EXPANDED_ABBR ]: {
+		getSlug: () => constants.FEATURE_ADVANCED_SEO_EXPANDED_ABBR,
+		getTitle: () => i18n.translate( 'Advanced SEO (Search Engine Optimisation) tools' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Advanced SEO (Search Engine Optimization) tools' )` *ES Score: 6*